Homework-1/index.js

```diff
@@ -0,0 +1,84 @@
+function loadReviews() {
+  if (!localStorage.getItem("reviews"))
+    localStorage.setItem("reviews", JSON.stringify([]));
+  const reviews = JSON.parse(localStorage.getItem("reviews"));
+  calculateAverage(reviews ? reviews : []);
+
+  document.getElementsByClassName("reviews")[0].innerHTML = "";
+  reviews.forEach((review, index) => {
+    const description = document.createElement("p");
+    description.innerText = review.description;
+
+    const rating = Number(review.rating);
+    const ratingStars = document.createElement("div");
+
+    for (let i = 0; i < 5; i++) {
+      const star = document.createElement("i");
+      star.classList.add("fas", "fa-star", "rating-star");
+      i < rating ? (star.style.color = "yellow") : (star.style.color = "white");
+      ratingStars.appendChild(star);
+    }
+
+    const removeButton = document.createElement("button");
+    removeButton.id = "button";
+    removeButton.onclick = () => removeReview(index);
+    removeButton.innerText = "Remove";
+
+    const reviewEl = document.createElement("div");
+    reviewEl.classList = ["review"];
+
+    reviewEl.appendChild(description);
+    reviewEl.appendChild(ratingStars);
+    reviewEl.appendChild(removeButton);
+
+    document.getElementsByClassName("reviews")[0].appendChild(reviewEl);
+  });
+}
+
+function handleSubmit(event) {
+  event.preventDefault();
+
+  const description = document.getElementById("description").value;
+  document.getElementById("description").value = "";
+
+  const rating = document.getElementById("rating").value;
+  for (let i = 0; i < 5; i++)
+    document.getElementsByClassName("star")[i].style.color = "white";
+
+  const newReview = { description: description, rating: rating };
+
+  let reviews = localStorage.getItem("reviews")
+    ? JSON.parse(localStorage.getItem("reviews"))
+    : [];
+  reviews.push(newReview);
+  localStorage.setItem("reviews", JSON.stringify(reviews));
```

Comment:
Similar to the comment I wrote for extracting the `document.getElementsByClassName("reviews")[0]`  statement to a variable, it would be smart to extract `localStorage.setItem("reviews", JSON.stringify(reviews));` to a function.
That way if in the future you need to switch from local storage to some other way of persisting items, you would need to make changes in a single function.